css/api-reference.scss

```diff
@@ -37,18 +56,14 @@ redoc {
     color: inherit !important;
   }
 
-  p, a, .menu-item-title {
+  p, a {
     font-family: $font !important;
   }
 
-  a, .menu-item-title {
+  a {
     color: $primary-links !important;
   }
 
-  .menu-item-title {
-    text-transform: none;
-  }
-
   // Disable request-type badges, see https://github.com/Rebilly/ReDoc/issues/61
   .operation-type {
```

Comment:
suggestion: Remove this hack to remove it. I'd really like to see these. It makes it much easier when browsing for specific keywords.

Reply:
Hmm, worth considering, but I'd vote for doing that in a separate PR.